AppInspector.CLI/Program.cs

```diff
@@ -25,103 +26,364 @@ static int Main(string[] args)
+            }
+
+            bool isValidFormat = validFormats.Any(v => v.Contains(fileFormatArgs));
+            if (!isValidFormat)
+            {
+                WriteOnce.Error(MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_ARG_VALUE, "-f"));
+                throw new OpException(MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_ARG_VALUE, "-f"));
+            }
+
+            //validate output is not empty if no file output specified
+            if (string.IsNullOrEmpty(options.OutputFilePath))
+            {
+                if (options.ConsoleVerbosityLevel.ToLower() == "none")
+                {
+                    WriteOnce.Error(MsgHelp.GetString(MsgHelp.ID.CMD_NO_OUTPUT));
+                    throw new Exception(MsgHelp.GetString(MsgHelp.ID.CMD_NO_OUTPUT));
+                }
+                else if (options.ConsoleVerbosityLevel.ToLower() == "low")
+                {
+                    WriteOnce.SafeLog("Verbosity set low.  Detailed output limited.", NLog.LogLevel.Info);
+                }
+            }
+            else
+            {
+                ValidFileWritePath(options.OutputFilePath);
+            }
+        }
+
+
+        /// <summary>
+        /// Ensure output file path can be written to
+        /// </summary>
+        /// <param name="filePath"></param>
+        static void ValidFileWritePath(string filePath)
+        {
+            try
+            {
+                File.WriteAllText(filePath, "");//verify ability to write to location
+            }
+            catch (Exception)
+            {
+                WriteOnce.Error(MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_FILE_OR_DIR, filePath));
+                throw new OpException(MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_FILE_OR_DIR, filePath));
+            }
+
+        }
+
+
+        #endregion
+
+
+        #region RunCmdsWriteResults
+
+        private static int RunAnalyzeCommand(CLIAnalyzeCmdOptions cliOptions)
+        {
+            AnalyzeResult.ExitCode exitCode = AnalyzeResult.ExitCode.CriticalError;
+
+            AnalyzeCommand command = new AnalyzeCommand(new AnalyzeOptions()
+            {
+                SourcePath = cliOptions.SourcePath,
+                CustomRulesPath = cliOptions.CustomRulesPath,
+                IgnoreDefaultRules = cliOptions.IgnoreDefaultRules,
+                AllowDupTags = cliOptions.AllowDupTags,
+                ConfidenceFilters = cliOptions.ConfidenceFilters,
+                MatchDepth = cliOptions.MatchDepth,
+                FilePathExclusions = cliOptions.FilePathExclusions,
+                ConsoleVerbosityLevel = cliOptions.ConsoleVerbosityLevel,
+                Log = cliOptions.Log
+            }); ;
+
+            AnalyzeResult analyzeResult = command.GetResult();
+            exitCode = analyzeResult.ResultCode;
+            ResultsWriter.Write(analyzeResult, cliOptions);
+
+            return (int)exitCode;
+        }
+
+
```

Comment:
Whitespace.

Reply:
Already added to commit with code clean up.